core-services/sanitize-prow-jobs/_config.yaml

```diff
@@ -5387,6 +5387,8 @@ groups:
     - openshift-release-infra-periodics.yaml
     - infra-periodics-origin-release-images.yaml
   build02:
+    jobs:
+    - openshift-oc-mirror-e2e-unit
```

Comment:
Why? We do not really want to have cluster-pinned jobs unless absolutely necessary.

Reply:
Relates to https://coreos.slack.com/archives/CBN38N3MW/p1639081903350000 as me and Hongkai Liu discovered, the last rotated cluster caused as segfault with nothing changed in the job or test.

Reply:
@petr-muller , how would you like this resolved? I can open a jira issue for the disparity between clusters or whatever is convenient for you.

Reply:
@sfxworks 
The issue on build04 might be caused by having troubles of scaling up which has been fixed now.
Could we try again without pinning the job onto b02 (just to see how the rehearsal goes)?

Reply:
@sfxworks @afflom we already said, several times, that we *do not* support handcrafted jobs. We won't block you from having them, but we will not support any kind of special treatment. Pinning a hand-crafted job to a cluster, because it (for any reason) is not able to run on an arbitrary build farm, is special treatment.

If you run a handcrafted job, *everything* is on you. We will not address differences between clusters, and we will not offer options. We told you that handcrafted jobs are not supported from the beginning, so if you decide to run it, we're fine with it, but we will not address any problems you encounter.